src/js/options.js

```diff
@@ -121,9 +121,9 @@ function loadOptions() {
     $("#settingsSuffix").css({"visibility": "hidden", "height": 0});
   }
 
-  // Show user's filters
+  // Set up option tabs
```

Comment:
Should this say something like "Populate Whitelisted Domains and Tracking Domains tabs" instead? Or this comment could just be removed, might be redundant given the clearly-named function invocations below.